### PR TITLE
Remove exit from code

### PR DESCRIPTION
**Pull request by williamdes**
### Description

I have no idea is this is a good enhancement but I wanted to remove as much 'exit;' to be able to mock the response function in order to test code and be able to remove some if TESTSUITE from codebase

Ref #15247

### CODE REVIEW

**Comment by williamdes** [comment]
Hi @mauriciofauth I hope this is a good idea, can you please review ?

**Review by MauricioFauth — APPROVED** [review]

**Comment by mauriciofauth** [comment]
I want to improve how we handle request and response in the future. Using a library that implements [PSR-7](https://www.php-fig.org/psr/psr-7/) or other HTTP library.

**Comment by mauriciofauth** [comment]
There is a lot of errors on Browserstack.

**Comment by williamdes** [comment]
A very much smaller change but if it passes and you approve @mauriciofauth that will be a small but good step to remove the 80+ exits in our code base

File: libraries/classes/Controllers/Database/StructureController.php [hunk 1]
```diff
@@ -425,7 +425,7 @@ public function multiSubmitAction(): void
                         /** @var ExportController $controller */
                         $controller = $containerBuilder->get(ExportController::class);
                         $controller->index();
-                        exit;
+                        return;
```
Comment (mauriciofauth) [inline libraries/classes/Controllers/Database/StructureController.php:428]: You also have to add a return after the `$this->multiSubmitAction();` call in the `index` method.

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

File: libraries/classes/Controllers/Database/StructureController.php (hunk 1 above)
Comment (mauriciofauth) [reply libraries/classes/Controllers/Database/StructureController.php:428]: This is not correct. :disappointed: 
Removing these exits will change the code flow. The best way to remove them is to extract each action into a dedicated route.

Comment (williamdes) [reply libraries/classes/Controllers/Database/StructureController.php:428]: I see, that will take me more time :)
I will have a try when possible

Comment (mauriciofauth) [reply libraries/classes/Controllers/Database/StructureController.php:428]: I've been extracting these `mult_submit` actions recently. They are confusing and difficult to read.

The other changes are good changes. :+1:

**Pull request merged**